### PR TITLE
Add test for TLA DFS invariant

### PR DESCRIPTION
**Pull request by jugglinmike**
This behavior was introduced after the Top-Level Await proposal reached
stage 3: https://github.com/tc39/proposal-top-level-await/pull/159

### CODE REVIEW

**Comment by jugglinmike** [comment]
@MylesBorins @codehag @guybedford Would you mind taking a look at this?

File: test/language/module-code/top-level-await/dfs-invariant.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                             dfs-invariant.js
+  .-----------------------------------+-------.
+  |                                   |       v
+  |                                   |       dfs-invariant-indirect_FIXTURE.js
+  |                               .---|----------------------'
+  v                               v   v
+  dfs-invariant-direct-1_FIXTURE.js   dfs-invariant-direct-2_FIXTURE.js
+            '--------.                            .--------'
+                     v                            v
+                     dfs-invariant-async_FIXTURE.js
```
Comment (rwaldron) [inline test/language/module-code/top-level-await/dfs-invariant.js:30]: I love when you make these <3

**Review by rwaldron — APPROVED** [review]

**Review by guybedford — APPROVED** [review]
Yes this captures the case, even though it's a slightly different example.

File: test/language/module-code/top-level-await/dfs-invariant.js (hunk 1 above)
Comment (jugglinmike) [reply test/language/module-code/top-level-await/dfs-invariant.js:30]: ,------------------. ->  @rwaldron  --.
    @jugglinmike -> | dfs-invariant.js |                  +-> Test262
                    '------------------' -> @guybedford --'

**Comment by jugglinmike** [comment]
Thanks, Guy! Thanks, Rick!

**Pull request merged**